src/registrar/management/commands/load_transition_domain.py

```diff
@@ -0,0 +1,463 @@
+                else:
+                    original_status = domain_status_dictionary[new_entry_domainName]
+                    # print(originalStatus)
+                    mapped_status = self.get_mapped_status(original_status)
+                    if mapped_status is None:
+                        logger.info("Unknown status: " + original_status)
+                        outlier_statuses.append(original_status)
+                    else:
+                        new_entry_status = mapped_status
+
+                if userId not in user_emails_dictionary:
+                    # this user has no e-mail...this should never happen
+                    if userId not in users_without_email:
+                        users_without_email.append(userId)
+                else:
+                    new_entry_email = user_emails_dictionary[userId]
+
+                # Check for duplicate data in the file we are
+                # parsing so we do not add duplicates
+                # NOTE: Currently, we allow duplicate domains,
+                # but not duplicate domain-user pairs.
+                # However, track duplicate domains for now,
+                # since we are still deciding on whether
+                # to make this field unique or not. ~10/25/2023
+                tempEntry_domain = next(
+                    (x for x in to_create if x.domain_name == new_entry_domainName),
+                    None,
+                )
+                tempEntry_domainUserPair = next(
+                    (
+                        x
+                        for x in to_create
+                        if x.username == new_entry_email
+                        and x.domain_name == new_entry_domainName
+                    ),
+                    None,
+                )
+                if tempEntry_domain is not None:
+                    if debug_on:
+                        logger.info(
+                            f"{termColors.WARNING} DUPLICATE Verisign entries found for domain: {new_entry_domainName} {termColors.ENDC}"  # noqa
+                        )
+                    if new_entry_domainName not in duplicate_domains:
+                        duplicate_domains.append(new_entry_domainName)
+                if tempEntry_domainUserPair is not None:
+                    if debug_on:
+                        logger.info(
+                            f"""{termColors.WARNING} DUPLICATE Verisign entries found for domain - user {termColors.BackgroundLightYellow} PAIR {termColors.ENDC}{termColors.WARNING}:  
+                            {new_entry_domainName} - {new_entry_email} {termColors.ENDC}"""  # noqa
+                        )
+                    if tempEntry_domainUserPair not in duplicate_domain_user_combos:
+                        duplicate_domain_user_combos.append(tempEntry_domainUserPair)
+                else:
+                    try:
+                        existingEntry = TransitionDomain.objects.get(
```

Comment:
what about just doing a filter here for the domain and then doing .exists() then do an if else block. Avoid using the try/except completely